src/path.zig

```diff
@@ -0,0 +1,26 @@
+const std = @import("std");
+const bun = @import("root").bun;
+
+const os = bun.Environment.os;
+const strings = bun.strings;
+
+pub fn isRelativePathCrossPlatform(path: []const u8) bool {
```

Comment:
this function implementation is
- better placed in `bun.path.Platform.`, that way it doesn't read `bun.Environment.os` and can use related utilities for slash direction and so on
- does not handle `./hello` since it has a length of 6, hitting the else, and then it never checks. i think the 2 => branch can be eliminated entirely
- incorrectly named since `hello.txt` is a relative path in most contexts (for example, in bash or the argument to `fs.open()`).